.travis/linux.win64.install.sh

```diff
@@ -3,10 +3,12 @@ set -e
 
 # First, install 32-bit deps
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-. $DIR/linux.win32.install.sh
+# Maybe find a better fix: see https://github.com/koalaman/shellcheck/wiki/SC1090
+# shellcheck disable=SC1090
```

Comment:
Can you please explain this in you PR rather than in code?  Disabling the shellcheck is perfectly adequate.  People can pull up `SC1090` on their own.